index.html

```diff
@@ -1,56 +1,65 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
-    <link rel="stylesheet" href="./style.css" type="text/css">
-    <title>Portfolio Homepage</title>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link href="https://fonts.googleapis.com/css?family=Poppins" rel="stylesheet" />
+  <link rel="stylesheet" href="./style.css" type="text/css" />
+  <title>Portfolio Homepage</title>
 </head>
+
 <body>
-    
   <!-- Navigation Bar Section -->
 
   <div class="wrapper">
     <div class="front-wrapper">
       <header>
-        <nav class="flex-nav">
-          <a href="https://yussufob.github.io/My-Portfolio-Part-II/" class="my-logo">Y-Bash</a>
-          <a href="https://yussufob.github.io/My-Portfolio-Part-II/" class="union-icon"><img src="./images/Union.svg" alt="list-icons"></a>
-          <ul class="menu-list">
-            <li><a href="https://yussufob.github.io/My-Portfolio-Part-II/">Portfolio</a></li>
-            <li><a href="https://yussufob.github.io/My-Portfolio-Part-II/">About</a></li>
-            <li>Contact</li>
+        <nav>
+          <div class="logo">Bash</div>
+          <div class="openMenu"><img src="./images/Union.svg" alt="hamburger"></div>
+          <ul class="mainMenu">
+            <li class="shieldOff1"><a href="#my-portfolio">Portfolio</a></li>
+            <li class="shieldOff2"><a href="#about-myself">About</a></li>
+            <li class="shieldOff3"><a href="#contact-me">Contact</a></li>
```

Comment:
- [ ] OPTIONAL: I suggest you improve the margins and spacing around the menu items, more separated like in the Figma.
- [ ] OPTIONAL: The font stylings do not match.  Your project is rendering "Arial".  It should be Poppins, font-weight 600, font-size, 32px, line-height 44px.   I suggest you kindly correct it.

If you implement these suggestions, your menu will look like the Figma! 💪